src/modules/ce/components/ProjectCePage.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import CommonTabs from '@/components/elements/CommonTabs';
+import PageTitle from '@/components/layout/PageTitle';
+import useSafeParams from '@/hooks/useSafeParams';
+import CreateOpportunityButton from '@/modules/ce/components/CreateOpportunityButton';
+import ProjectOpportunitiesTable from '@/modules/ce/components/ProjectOpportunitiesTable';
+import ProjectReferralsTable from '@/modules/ce/components/ProjectReferralsTable';
+
+interface Props {}
```

Comment:
Should we make some changes to the established routes? This is currently at `/project/:id/opportunities`, and with the introduction of the CommonTabs we now have
- `/project/:id/opportunities#opportunities` 
- `/project/:id/opportunities#referrals` 

Maybe it would be nice to change the base URL from `/project/:id/opportunities` to `/project/:id/ce`? That gives us
- `/project/:id/ce#opportunities` 
- `/project/:id/ce#referrals`

Reply:
Yes I think we should change this page route to be "ce" and use `Coordinated Entry` as the page title. thanks!